src/lib/api.ts

```diff
@@ -85,55 +85,57 @@ export async function getSponsors() {
   }
 
   const response = await octokit().graphql(`
-    query {
-      user(login: "rfay") {
-        ... on Sponsorable {
-          sponsors(first: 100) {
-            totalCount
-            nodes {
-              ... on User {
-                login
-                url
-                avatarUrl
-              }
-              ... on Organization {
-                login
-                url
-                avatarUrl
-              }
+    query CombinedSponsors {
+      org: organization(login: "ddev") {
+        sponsors(first: 100) {
+          nodes {
+            ... on User {
+              login
+              url
+              avatarUrl
+            }
+            ... on Organization {
+              login
+              url
+              avatarUrl
             }
           }
         }
       }
-      organization(login: "ddev") {
-        ... on Sponsorable {
-          sponsors(first: 100) {
-            totalCount
-            nodes {
-              ... on User {
-                login
-                url
-                avatarUrl
-              }
-              ... on Organization {
-                login
-                url
-                avatarUrl
-              }
+      user: user(login: "rfay") {
```

Comment:
are you sure we want to change the user here?

Reply:
Before we had the `ddev` org, some people had started sponsoring the project via the `rfay` account. This was also accounted for previously in the code here, but the logic was different. There are still 17 people and orgs sponsoring DDEV via the rfay account.